helm/mailhog/templates/ingress.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
Ingress are for connections from outside of the namespace (and Openshift uses routes anyway) so I don't think that you need this template